setup.py

```diff
@@ -26,13 +27,65 @@
 INSTALL_REQUIRES = ['xarray>=0.12.0', 'dask', 'numpy>=1.16']
 PYTHON_REQUIRES = '>=3.6'
 
-ext_cape_ml = Extension(name = 'xcape.CAPE_CODE_model_lev',
+try:
+        # figure out which compiler we're going to use
+        compiler = get_default_fcompiler()
+        # set some fortran compiler-dependent flags
+        f90flags = []
+        if compiler == 'gnu95':
+            f90flags.append('-fno-range-check')
+            f90flags.append('-ffree-form')
+        elif compiler == 'intel' or compiler == 'intelem':
+            f90flags.append('-132')
+        #  Set aggressive optimization level
+        f90flags.append('-O3')
+        #  Suppress all compiler warnings (avoid huge CI log files)
+        f90flags.append('-w')
+except :
+        print('No Fortran compiler found, not building the RRTMG_LW radiation module!')
```

Comment:
This doesn't seem like the right error message

Reply:
This section is also over-indented.

Reply:
I guess I would recommend just removing this `try` / `except` block and letting setup raise an error if it can't find a fortran compiler. This package can't be built without it.